edb/api/errors.txt

```diff
@@ -140,5 +140,6 @@
 0x_FF_02_01_00   QueryArgumentError
 0x_FF_02_01_01   MissingArgumentError
 0x_FF_02_01_02   UnknownArgumentError
+0x_FF_02_01_03   InvalidArgumentValueError
```

Comment:
You seem to be using this new error for both TypeError and ValueError.  Should we add InvalidArgumentTypeError and carefully separate them?